문제10505.py

```diff
@@ -0,0 +1,13 @@
+result2print = []
+for x in range(int(input())) :
+    N = int(input())
+    list_num = list( map( int, input().strip().split(sep=" ") ) )
+    list_num.sort()
+    mean_value = int(sum(list_num)/N)
```

Comment:
좋은 풀이입니다 👍 
```suggestion
    mean_value = sum(list_num)/N
```
`int`는 버림 연산으로 가까운 정수를 반환하기 때문에, 정확한 평균 값을 구하시려면 `int` 함수를 적용하지 않는 것이 맞습니다.